### DISCUSSION TITLE
Microsoft Edge - Manifest v3 - Chrome.storage.local does not appear to be working

### DISCUSSION THREAD

**Discussion by HowardRicketts1** [microsoft/MicrosoftEdge-Extensions#75] [Discussions]
I am running version Microsoft Edge Version 109.0.15.18.55 (Official build) (64 bit) and have spent over a week trying to get a simple extension to write a string to Local storage using chrome.storage.local.set but don't believe it is being written.  My extension is attempting to store a hard-coded string 'happiness' against a property 'wish' using chrome.storage.local.set but when I look at Developer Tools>Application>Local Storage there is no properties listed and when I attempt to retrieve the property 'undefined' is returned.

Can someone please provide me with a simple working example of the chrome.storage.local.set and chrome.storage.local..get APIs in a simple Microsoft Edge extension.   

 I have also attempted to migrate the Stylizr example from Manifest v2 to Manifest v3 but failed to get it working even though it does not show any errors.

**Comment by HowardRicketts1** [comment]
I eventually managed to get this working although I havent found a satisfactory way of using DevTools with MS Edge Extensions.  How can I close this Conversation?

**Reply by sivsouvamMSFT** [reply]
Hey @HowardRicketts1, maybe you can share, what worked out for you. It will be helpful for community members. This conversation won't get closed.

**Reply by starsoccer** [reply]
Also having this issue intermittently. I thought it was an issue with an older version of edge as updating resolved it but after a reboot the issue came back

**Comment by Nosa-Drexx** [comment]
Also experiencing the same thing, how did you resolve the issue @HowardRicketts1

**Comment by ManikanthMSFT** [comment]
Hey @HowardRicketts1, we believe it would be beneficial for everyone if you could share what has worked well for you. Your insights could be incredibly valuable to other community members.

**Comment by FMaz008** [comment]
Do you have the storage permission in your manifest v3?
